store/tikv/gc_worker.go

```diff
@@ -465,6 +510,9 @@ func resolveLocks(ctx goctx.Context, store *tikvStore, safePoint uint64, identif
 func doGC(ctx goctx.Context, store *tikvStore, safePoint uint64, identifier string) error {
 	gcWorkerCounter.WithLabelValues("do_gc").Inc()
 
+	store.gcWorker.saveSafePoint(gcSavedSafePoint, safePoint)
+	time.Sleep(gcSafePointCacheInterval)
```

Comment:
Add comment herer.
I think it's better to getTS() again and ensure there are at least `gcSafePointCacheInterval` escaped, depending on local clock is unreliable.

Reply:
Good Insight! Thanks~